package.json

```diff
@@ -30,5 +30,11 @@
     "topojson-client": "^3.0.0",
     "vanilla-framework": "^1.6.5",
     "watch-cli": "^0.2.2"
+  },
+  "devDependencies": {
```

Comment:
I'd move it all to `dependencies`.
It's not a project that is going to be installed as npm module, so difference between dependencies and devDependencies is not needed and in long term may only be confusing (should something be dev or not).